src/classes/presentation_controller.vala

```diff
@@ -2218,5 +2233,37 @@ namespace pdfpc {
             rect = c.main_view.convert_poppler_rectangle_to_gdk_rectangle(area);
         }
 #endif
+
+        /**
+         * Timeout id to delay automatic hide of the presentation window
+         */
+        private uint auto_hide_timeout_id = 0;
+
+        private void handle_auto_hide_event() {
+            if (!Options.auto_hide)
+                return;
+
+            if (!this.presenter.is_active && !this.presentation.is_active) {
```

Comment:
Why would anyone want to switch focus from the presenter to the presentation window?